app.rb

```diff
@@ -63,5 +67,17 @@ class GrootMerchService < Sinatra::Base
     disable :unsecure
   end
 
+  configure do
+    scheduler = Rufus::Scheduler.new
+    logger = Logger.new STDOUT
+    logger.level = Logger::DEBUG
+    logger.datetime_format = '%Y-%m-%d %H:%M:%S '
+    jobs = Jobs.new
+    logger.info jobs.restock
```

Comment:
Is there a reason we have to do it on startup, rather than just let the scheduler do it?